google/ads/google_ads/oauth2.py

```diff
@@ -0,0 +1,107 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""A set of functions to help initialize OAuth2 credentials."""
+
+
+import functools
+
+from google.oauth2.service_account import Credentials as ServiceAccountCreds
+from google.oauth2.credentials import Credentials as InstalledAppCredentials
+from google.auth.transport.requests import Request
+
+from google.ads.google_ads import config
+
+_SERVICE_ACCOUNT_SCOPES = ['https://www.googleapis.com/auth/adwords']
+_DEFAULT_TOKEN_URI = 'https://accounts.google.com/o/oauth2/token'
```

Comment:
Endpoint may have changed. See https://github.com/golang/oauth2/issues/310

Reply:
Both URLs seem to work, let's investigate further whether a change is necessary.

Reply:
We couldn't find further evidence this is an issue so I'm resolving.